backend/app/__init__.py

```diff
@@ -1,9 +1,22 @@
-from flask import (
-    Flask
-)
+import os
+from flask.helpers import url_for
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+from os import error
+from flask import Flask, request, jsonify, url_for, redirect
+from . emotionDetection import getEmotion 
+import tensorflow as tf
+
+import base64
+import datetime
+from urllib.parse import urlencode
+import requests
 
 app = Flask(__name__)
 
+CLIENT_ID = os.environ.get('CLIENT_ID')
+CLIENT_SECRET = os.environ.get('CLIENT_SECRET')
+
 @app.route("/")
 def home():
     return "Works"
```

Comment:
It might make more sense to send codes instead of `works` here. `200` is more appropriate for example.

Reply:
Done 🚀